sigmastate/src/test/scala/sigmastate/serialization/DeserializationResilience.scala

```diff
@@ -255,6 +288,12 @@ class DeserializationResilience extends SerializationSpecification
       ValueSerializer.deserialize(reader(ValueSerializer.serialize(expr), maxTreeDepth = 3))
```

Comment:
Please make sure the thrown exception comes from TypeSerializer. I.e. that the test is make sense.

Reply:
Updated in new version.